app/assets/javascripts/action_plan.js

```diff
@@ -0,0 +1,9 @@
+document.addEventListener("turbolinks:load", function() {
+  $("input[name='action-plan_prefix']").keyup(function(event) {
+    const prefix = event.target.value.trim();
+
+    $("h3.action-plan_heading > span").each(function(index) {
+      $(this).text(`${prefix}${prefix == "" ? "" : index + 1 } `);
```

Comment:
I noticed that it is using `==` instead of `===` would it need to coerce the prefix to ""?

Reply:
I wonder if you should also add the (dot) when you append the index?
```suggestion
      const suffix = prefix ? "" : `.${index+1}`
      $(this).text(`${prefix}${suffix } `);
```

Reply:
> I wonder if you should also add the (dot) when you append the index?

@buys-fran I was wondering this as well, while looking at https://github.com/fastruby/points/issues/23 I assumed that the expected input would be a prefix that looked like `3.2.`. I am ok with either. It's probably a better user experience to add the dot, what do you think?

Reply:
I agree, it does seem more user friendly :-)